packages/react-components/src/components/MessageThread.tsx

```diff
@@ -360,14 +373,107 @@ const memoizeAllMessages = memoizeFnAll(
       disableEditing
     };
 
+    /* @conditional-compile-remove(dlp) */
+    // Same logic as switch statement, if statement for conditional compile
+    if (message.messageType === 'blocked') {
```

Comment:
conditional-compile-remove  doesn't seem to work on `switch` cases, so breaking out to `if` statement. Have same logic as `case 'chat':`